### PR TITLE
Add Titles to all ODC Screenshots

### PR DESCRIPTION
**Pull request by nkakkar81**
JIRA - https://issues.redhat.com/browse/RHDEVDOCS-1736

### CODE REVIEW

**Comment by nkakkar81** [comment]
@Preeticp - Request you to please review this PR. Thanks

**Comment by Preeticp** [comment]
Hi @nkakkar81 there is this one assembly which has a figure: Creating applications using the Developer perspective: https://odc-rhdevdocs-1736--ocpdocs.netlify.app/openshift-enterprise/latest/applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.html
I just hope the numbering does not go awry cause it is an assembly.

**Comment by nkakkar81** [comment]
@Preeticp - Have added the title to the assembly you pointed in your comment. PTAL and review. If you are ok with the changes, request you o please provide LGTM. Thanks!

File: applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc [hunk 1]
```diff
@@ -7,7 +7,8 @@ toc::[]
 
 The *Developer* perspective in the web console provides you the following options from the *Add* view to create applications and associated services and deploy them on {product-title}:
 
-image::odc_add_view.png[Add View]
+.Add View
```
Comment (bergerhoffer) [inline applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc:10]: All titles added throughout this PR should be sentence case.

**Review by bergerhoffer — COMMENTED** [review]
LGTM other than the titles need to be reviewed and some of them need to be updated to be sentence case, instead of title case.

File: applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc (hunk 1 above)
Comment (nkakkar81) [reply applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc]: @bergerhoffer - I have made the changes suggested by you, request you to please review and let me know if you are ok with the changes. Thanks

**Comment by bergerhoffer** [comment]
LGTM now!

**Pull request merged**